tests/ctests/IsdTests.cpp

```diff
@@ -0,0 +1,679 @@
+#include <string>
+#include <fstream>
+#include <streambuf>
+
+#include "gtest/gtest.h"
+
+#include "ale.h"
+#include "Isd.h"
+#include "Util.h"
+
+void ASSERT_DOUBLE_VECTOR_EQ(std::vector<double> v1, std::vector<double> v2) {
+  if (v1.size() != v2.size()) {
+    EXPECT_TRUE(::testing::AssertionFailure() << "The two input vectors are different in size");
```

Comment:
I would assert true here. You could seg_fault in the for loop below if this is false.